.azurepipelines/publish-maven-1ES.yml

```diff
@@ -0,0 +1,101 @@
+trigger: none
+name: $(date:yyyyMMdd)$(rev:.r)
+variables:
+- name: BUILD_DIR

```

Comment:
Where do we use it?

Reply:
It's part of publishing script. It is used to specify the local path the files will be stored. See https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/AppCenterReactNativeShared/android/build.gradle#L64